administration/src/util/translations/de.json

```
@@ -0,0 +1,72 @@
+{
+  "application": {
+    "personalData": "Persönliche Daten",
+    "forenames": "Vorname(n)",
+    "surname": "Nachname",
+    "address": "Adresse",
+    "street": "Straße",
+    "houseNumber": "Hausnummer",
+    "addressSupplement": "Adresszusatz",
+    "postalCode": "Postleitzahl",
+    "location": "Ort",
+    "country": "Land",
+    "dateOfBirth": "Geburtsdatum",
+    "emailAddress": "E-Mail-Adresse",
+    "region": "Zuständige Behörde",
+    "applicationDetails": "Antragsdetails",
+    "cardType": "Antrag auf",
+    "applicationType": "Art des Antrags",
+    "wantsDigitalCard": "Ich beantrage eine digitale Ehrenamtskarte",
+    "wantsPhysicalCard": "Ich beantrage eine physische Ehrenamtskarte",
+    "blueCardJuleicaEntitlement": "Ich bin Inhaber:in einer JuLeiCa (Jugendleiter:in-Card)",
+    "juleicaNumber": "Kartennummer",
+    "juleicaExpiration": "Karte gültig bis",
+    "copyOfJuleicaFront": "Kopie der Karte (1)",
+    "copyOfJuleicaBack": "Kopie der Karte (2)",
+    "blueCardWorkAtOrganizationsEntitlement": "Ich engagiere mich ehrenamtlich seit mindestens zwei Jahren freiwillig mindestens fünf Stunden pro Woche oder bei Projektarbeiten mindestens 250 Stunden jährlich",
+    "workAtOrganization": "Arbeit bei Organisation oder Verein",
+    "organization": {
+      "title": "Angaben zur Organisation",
+      "name": "Name der Organisation bzw. des Vereins",
+      "category": "Einsatzgebiet"
+    },
+    "organizationContact": {
+      "title": "Kontaktperson der Organisation",
+      "name": "Vor- und Nachname der Kontaktperson in der Organisation",
+      "telephone": "Telefon",
+      "email": "E-mail",
+      "hasGivenPermission": "Die Kontaktperson hat der Weitergabe ihrer Daten zum Zwecke der Antragsverarbeitung zugestimmt und darf zur Überprüfung kontaktiert werden"
+    },
```

:wrench: I really dislike the `getTranslationKey()` in `JsonFieldArray` as I think its quite hacky and error prone in the future. Getting rid of the namespace would make it redundant.
```suggestion
    "organization": "Angaben zur Organisation",
    "organizationName": "Name der Organisation bzw. des Vereins",
    "organizationCategory": "Einsatzgebiet"
    "organizationContact": "Kontaktperson der Organisation",
    "organizationContactName": "Vor- und Nachname der Kontaktperson in der Organisation",
    ...
    },
```

---

if we decide we want to do this, yes. I will create one, then we can discuss this in the grooming.

---

Thanks!

---

https://github.com/digitalfabrik/entitlementcard/issues/1770